src/no-inline-script.ts

```diff
@@ -0,0 +1,38 @@
+import { Rule } from 'eslint';
+import { Node, CallExpression } from 'estree';
+
+const noInlineScript: Rule.RuleModule = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Disallow the injection of scripts to the HTML directly',
+      category: 'Security',
+      recommended: false,
+    },
+    messages: {
+      noInlineScript: 'Adding scripts to HTML directly is not allowed',
+    },
+  },
+  create(context: Rule.RuleContext) {
+    return {
+      CallExpression(n: Node) {
+        const node = n as CallExpression;
+        if (
+          node.callee.type === 'MemberExpression' &&
+          node.callee.property.type === 'Identifier' &&
+          node.callee.property.name === 'createElement' &&
+          node.arguments[0] &&
+          node.arguments[0].type === 'Literal' &&
+          node.arguments[0].value === 'script'
```

Comment:
not bulletproof, but best effort could include `.toLowerCase`

Reply:
done!